onegov/election_day/layouts/archive.py

```diff
@@ -0,0 +1,32 @@
+from onegov.election_day.collections import SearchableArchivedResultCollection
+from onegov.election_day.layouts import DefaultLayout
+from cached_property import cached_property
+from onegov.election_day.models import ArchivedResult
+
+
+class ArchiveLayout(DefaultLayout):
+
+    def __init__(self, model, request):
+        super().__init__(model, request)
+
+    @cached_property
+    def menu(self):
+
+        return [
+            (label, self.link_for(abbrev), self.model.item_type == abbrev)
+            for abbrev, label in ArchivedResult.types_of_results[0:2]
+        ]
+
+    @cached_property
+    def tab_menu_title(self):
+        mapping = {k: v for k, v in ArchivedResult.types_of_results}
+        return mapping.get(self.model.item_type, 'ERROR!')
```

Comment:
If this error can actually occur, an exception would be better. Exceptions are tracked on our side and we would see them. A user seeing "Error!" is something we'd miss.